src/products/mozilla_org/manage_mozillaorg.md

```diff
@@ -0,0 +1,95 @@
+
+Once the PR passes the review, L10n PM will approve and merge the PR to master.
+
+## Managing the project
+
+### Updating status in Pontoon
+
+* After Pontoon syncs with the repository, the newly added file will appear on the dashboard of a locale.
+* Go to the [project admin page](https://pontoon.mozilla.org/admin/projects/mozillaorg/).
+Scroll to the **Linked tags** section to set priority to the new file by following [these steps](https://github.com/mozilla-l10n/documentation/blob/master/src/tools/pontoon/adding_new_project.md#tags).
+* Set a new deadline for the project if applicable.
+* Save the changes made.
+* If necessary, set a page-level deadline by following the instructions on the [Resource page](https://pontoon.mozilla.org/a/base/resource/). Select the file, then add the deadline near the bottom of the page.
+
+### Email communication
+
+Sometimes a page is tied to a release. Either the time is short or we want to have as many languages complete as possible, an email to the communities is necessary soon after the page is ready on Pontoon. In the email, include the following information:
+
+* File name along with word and string counts.
+* Priority locales if any.
+* Deadlines if they are different between priority locales.
+* Specify the locales that are covered by Mozilla staff, as long as they are working in Pontoon.
+* Demo URL for testing.
+* Special instruction for this particular file to watch out for if any.
+
+Send the email to the web [project mailing list](https://groups.google.com/g/mozilla.dev.l10n.web). Forward the same information to the Mozilla copywriters.
+
+### What's New Page (WNP)
+
+Every month, there will be a WNP tied to the release of Firefox for desktop. A bug like [this](https://bugzilla.mozilla.org/show_bug.cgi?id=1649046) will be filed. The l10n PM will be notified on the deadline when the locale list is due. Ideally, the page should be ready for localization between 10-14 days. It has rarely been the case. Communicate to the communities as soon as the content is available and update Pontoon with the deadline.
+
+The list of completed locales is due to the release team on Friday, 10 days prior to the release date. If the release specific WNP can’t be ready by deadline, the [generic WNP list](https://github.com/mozmeao/www-l10n/blob/master/metadata/firefox/whatsnew/whatsnew.json) will do.
+
+## Staging and Production
+
+Pontoon syncs with the repository every 10 minutes. GitHub makes a production push every 5 minutes. Unless a page is tied to a release on a particular day, chances are, the localization review is done on Production.
+
+When a localized string contains Fluent syntax error, it will be rejected in the  production push, thus the string appears in English.
+
+### Page Activation
+
+It is not unusual when a localized page is mixed with some English strings at a given moment. Here are some scenarios:
```

Comment:
I have some troubles following the first sentence, I assume it means "it is not unusual for a page to have mixed content"?